File: problem_twenty_one.py
# count prime number  from low to high number.
low = int(input("Enter low number:"))
high = int(input("Enter upper  number:"))

def MyFunction(low,high):
    count = 0
    for i in range(low,high+1):
        if i > 1:
            for j in range(2,i):
                if i%j== 0 :
                   break
            else:
               count+=1
    return count

print("count = ",MyFunction(low,high))             


    


                   


   
    
    

File: problem_twenty_one.js
// count prime  from low to high
var low = 10
var high = 50
function myFunction(low,high){
    var count = 0
    for (var i = low ; high+1 > i ; i++){
        var  val = true
        if(i >1){
            for(var j = 2 ; i > j;j++){
                if( i% j===0){
                    val  = false
                    break           
                }     
            }
            if(val){
               count+=1
            }            
        }  
    }
    return count  
}
console.log(myFunction(low,high))
